### PR TITLE
Andres ruiz tarea 7

### PR DESCRIPTION
**Pull request by andresCruiz**

### CODE REVIEW

File: Andres-ruiz/tarea-3/oddOrEven.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+function oddOrEven(array) {
```
Comment (emolinaglobant) [inline Andres-ruiz/tarea-3/oddOrEven.js]: @andresCruiz , make this  a one line solution

File: Andres-ruiz/tarea-3/largestPairSum.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+ function largestPairSum (numbers) {
```
Comment (emolinaglobant) [inline Andres-ruiz/tarea-3/largestPairSum.js]: @andresCruiz , make this  a one line solution

File: Andres-ruiz/tarea-3/DuplicateSandwich.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+function duplicateSandwich(arr) {
```
Comment (emolinaglobant) [inline Andres-ruiz/tarea-3/DuplicateSandwich.js:1]: @andresCruiz , make this  a one line solution

File: Andres-ruiz/tarea-4/js/index.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const displayValorAnterior = document.getElementById('valor-anterior');
+const displayValorActual = document.getElementById('valor-actual');
+const botonesNumeros = document.querySelectorAll('.numero');
+const botonesOperadores = document.querySelectorAll('.operador');
+
+const display = new Display(displayValorAnterior, displayValorActual);
+
+botonesNumeros.forEach(boton => {
```
Comment (emolinaglobant) [inline Andres-ruiz/tarea-4/js/index.js]: @andresCruiz , use english as main language for programming

**Review by emolinaglobant — CHANGES_REQUESTED** [review]
@andresCruiz , i don't see the unit tests for "tarea-7"

also why tarea-3 so late?

**Comment by emolinaglobant** [comment]
@andresCruiz where are the unit tests?

File: Andres-ruiz/tarea-4/js/index.js (hunk 4 above)
Comment (andresCruiz) [reply Andres-ruiz/tarea-4/js/index.js]: this  solution is in another branch

**Comment by emolinaglobant** [comment]
@SebastianRuedaGlobant , you did not attend last comment

**Pull request closed without merging**